### PR TITLE
Nebula12 keyboard

### PR DESCRIPTION
**Pull request by yiancar**
Sibling to Nebula65

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nebula12/keymaps/default/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+The default keymap for Nebula12. VIA support disabled.
+=========================================================
```
Comment (fauxpark) [inline keyboards/nebula12/keymaps/default/readme.md:2]:
```suggestion
# The default keymap for Nebula12. VIA support disabled.
```

File: keyboards/nebula12/keymaps/via/readme.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The default keymap for Nebula12. VIA support enabled.
+=========================================================
```
Comment (fauxpark) [inline keyboards/nebula12/keymaps/via/readme.md:2]:
```suggestion
# The default keymap for Nebula12. VIA support enabled.
```

File: keyboards/nebula12/readme.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Nebula65
+=========
```
Comment (fauxpark) [inline keyboards/nebula12/readme.md:2]:
```suggestion
# Nebula65
```

File: keyboards/nebula12/nebula12.h [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#define XXX KC_NO
+
+#include "quantum.h"
+#include "../wilba_tech/wt_rgb_backlight_keycodes.h"
+#include "via.h"
```
Comment (fauxpark) [inline keyboards/nebula12/nebula12.h:22]: Are these really needed here?

File: keyboards/nebula12/readme.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+This is a standard fixed layout 12 key PCB. It supports VIA, full per-key RGB and underglow RGB.
+
+Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)\
+Hardware Supported: A 65% keyboard with STM32F072CB\
+Hardware Availability: https://spaceholdings.net/
```
Comment (fauxpark) [inline keyboards/nebula12/readme.md:8]:
```suggestion
* Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
* Hardware Supported: A 65% keyboard with STM32F072CB
* Hardware Availability: https://spaceholdings.net/
```

File: keyboards/nebula12/rules.mk [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# MCU name
+MCU = STM32F072
+BOARD = ST_STM32F072B_DISCOVERY
```
Comment (fauxpark) [inline keyboards/nebula12/rules.mk:3]:
```suggestion

BOARD = ST_STM32F072B_DISCOVERY
```

File: keyboards/nebula12/nebula12.h (hunk 4 above)
Comment (yiancar) [reply keyboards/nebula12/nebula12.h]: `#include "../wilba_tech/wt_rgb_backlight_keycodes.h"` Is needed for the wilbacode inswitch keycodes.

Now fixed

File: keyboards/nebula12/rules.mk [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+WS2812_DRIVER = spi                # Underglow RGB Driver
+
+CIE1931_CURVE = yes
+
+# project specific files
+SRC =	keyboards/wilba_tech/wt_main.c \
+		keyboards/wilba_tech/wt_rgb_backlight.c \
+		drivers/issi/is31fl3731.c \
+		quantum/color.c \
+		drivers/chibios/i2c_master.c
```
Comment (drashna) [inline keyboards/nebula12/rules.mk:34]:
```suggestion
SRC +=  keyboards/wilba_tech/wt_main.c \
        keyboards/wilba_tech/wt_rgb_backlight.c \
        drivers/issi/is31fl3731.c \
        quantum/color.c
QUANTUM_LIB_SRC =+ drivers/chibios/i2c_master.c
```

**Comment by yiancar** [comment]
Fixed the merge conflict

**Pull request merged**